crates/repo/src/lib.rs

```diff
@@ -279,4 +220,40 @@ mod tests {
             })
         );
     }
+
+    #[tokio::test]
+    async fn routes() {
+        let listener = TcpListener::bind("127.0.0.1:8000".parse::<SocketAddr>().unwrap()).unwrap();
+        let addr = listener.local_addr().unwrap();
+
+        tokio::spawn(async move {
+            axum::Server::from_tcp(listener)
+                .unwrap()
+                .serve(app().into_make_service())
```

Comment:
What if port `8000` is already used?

Let's do exactly what we already do in main with no custom logic
https://github.com/profianinc/drawbridge/blob/19f8f307bcd7b30f921aa3d96a2876bca0cb73c6/src/main.rs#L14